From 0ff20f70d2f0bd734ad222eacd092459392961b7 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 9 Oct 2024 18:23:43 +0200
Subject: [PATCH 1279/1695] dts: stm32: remove IPCC2 node for stm32mp23x Socs

The IPCC2 is not present on the stm32mp23x Socs

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git a7db085ba4ef8495f4e8ff0ae4cc5f9f85eb461c dts: stm32: remove IPCC2 node for stm32mp23x Socs]
Change-Id: Id62d298e650866167a2f607e0489441af10f7898
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/410275
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 4eb724a58..9d4e362d8 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -3237,12 +3237,6 @@
 			clocks = <&rcc CK_BUS_HSEM>;
 		};
 
-		ipcc2: mailbox@46250000 {
-			compatible = "st,stm32mp25-ipcc";
-			reg = <0x46250000 0x400>;
-			clocks = <&rcc CK_BUS_IPCC2>;
-		};
-
 		stgenc: stgen@48080000 {
 			compatible = "st,stm32mp25-stgen";
 			reg = <0x48080000 0x1000>;
-- 
2.47.2

